### PR TITLE
Cart Functionality

### LINKED ISSUE #23: Cart page
- Should be able to add items to cart
- Move wishlisted items to cart
- Should be able to increase / decrease quantity of items in cart

### PR DESCRIPTION
**Pull request by ttahm3d**
- All the cart functionality works
- Replaced header with icon links

Closes issue #23

### CODE REVIEW

**Review by aishwarya-bm — APPROVED** [review]
code looks great with many reusable parts & easy to add new code. Liked how the cart dispatch actions are handled.

File: src/components/Badge/Badge.module.css [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  font-size: 0.75rem;
+  font-weight: 600;
+  top: 0.25rem;
+  right: 0.5rem;
+  min-width: 1.3rem;
+  min-height: 1rem;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  padding: 5%;
```
Comment (nishasen) [inline src/components/Badge/Badge.module.css:33]: You can add util class here

File: src/context/cart/Context.jsx [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+      try {
+        const res = await axios.post(
+          "/api/user/cart",
+          { product },
+          headerConfig
+        );
+        if (res.status === 201) {
+          cartDispatch({
+            type: "ADD_TO_CART",
+            payload: res?.data?.cart,
```
Comment (nishasen) [inline src/context/cart/Context.jsx:55]: I like this part of your code

**Review by nishasen — APPROVED** [review]
I like your well structured code and reusable components.

**Pull request merged**